### PR TITLE
Backport mu-wpcom-plugin 2.0.26, jetpack 13.2-a.11 Changes

### PR DESCRIPTION
**Pull request by samiff**
## Proposed changes:

Backports changes for mu-wpcom-plugin 2.0.26, jetpack 13.2-a.11

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/a
## Does this pull request change what data or activity we track or use?

No
## Testing instructions:

Proofread changes.

### CODE REVIEW

File: projects/plugins/jetpack/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Jetpack",
-	"version": "13.2.0-a.10",
+	"version": "13.2.0-a.12",
```
Comment (tbradsha) [inline projects/plugins/jetpack/package.json:3]: Should this be `13.2.0-a.11`?

**Review by tbradsha — APPROVED** [review]
As far as I can tell this looks good, with one question inline.

Marking as approved.

**Review by tbradsha — DISMISSED** [review]
Except now I see a bunch of merge conflicts, so those need to be resolved first.

File: projects/plugins/jetpack/package.json (hunk 1 above)
Comment (kraftbj) [reply projects/plugins/jetpack/package.json:3]: No, trunk is at even intervals and the released weeklies use odd numbers so trunk isn't seen as older than an older released version.

Comment (tbradsha) [reply projects/plugins/jetpack/package.json:3]: Shows how little I know about the release processes!

**Review by kraftbj — DISMISSED** [review]

**Review by kraftbj — APPROVED** [review]

**Pull request merged**